shell/platform/darwin/ios/framework/Source/accessibility_bridge.h

```diff
@@ -45,7 +46,7 @@ class AccessibilityBridge;
  * The accessibility bridge that this semantics object is attached to. This
  * object may use the bridge to access contextual application information.
  */
-@property(nonatomic, readonly) shell::AccessibilityBridge* bridge;
+@property(nonatomic, readonly) fml::WeakPtr<shell::AccessibilityBridge> bridge;
```

Comment:
Let's add a one-liner in the doc comment: "Weak pointer is used because the bridge's lifecycle is owned by PlatformView".

Reply:
Done.